### ISSUE TITLE
Update to work with Laravel 8.0?

### ISSUE THREAD

**Issue by kenwilliams5** [christophrumpel/laravel-factories-reloaded#62]
Will this package be updated to work with Laravel 8.0?

Thanks...

**Comment by christophrumpel** [comment]
Hey, yeah I'm already at it. It just turned out to be much more work than expected. Hope I can finish it this week.

**Comment by kenwilliams5** [comment]
Thank you!! I wish I knew enough about it to help!

**Comment by christophrumpel** [comment]
No worries. I'm getting closer 👍

**Comment by tmgbedu** [comment]
Any Updates ?

**Comment by christophrumpel** [comment]
Hey, sorry not yet. Client work keeping me away currently of my open source work.

**Comment by christophrumpel** [comment]
Getting closer now. Working on the last failing tests.

**Comment by ap-coder** [comment]
Looking forward to your new package do you by chance have a timeframe as we could use it now. Keep up the great work.

**Comment by christophrumpel** [comment]
Thax @ap-coder. Couldn't hold the first two time-frames so I'm careful now :-) New plan is till the end of the week. 🤞

**Comment by christophrumpel** [comment]
Hey, made some good progress today with some help from @nuernbergerA. Anyone up to test it on their application when I push a new release the next days?

**Comment by GuidoHendriks** [comment]
@christophrumpel Would be happy to  😄

**Comment by christophrumpel** [comment]
@GuidoHendriks That would be great. I pushed all the changes to the branch `support-laravel-8`. Can you try this in your L8 application?

**Comment by bobo-le** [comment]
@christophrumpel works as expected in 200 tests

**Comment by christophrumpel** [comment]
Thanks a lot @bobo-le. @GuidoHendriks did you also already had the chance to test it?
I need to update the docs but I think then we are good to go for a first new major release. Since the L8 changes of factories the new release will only work with L8.

**Comment by GuidoHendriks** [comment]
@christophrumpel Currently in the progress of upgrading my project, as it was previously blocked by the lack of support in this package. Will let you know asap.

**Comment by christophrumpel** [comment]
Ah I see, perfect. Thanks

**Comment by GuidoHendriks** [comment]
@christophrumpel All good with my 1100+ tests.

Might however switch back to Laravel's factories with some php doc comments to solve autocompletion.

**Comment by christophrumpel** [comment]
Thanks @GuidoHendriks.

Yeah this is a valid option too. For now, I want to provide L8 with the package so that everyone can use it if they want. The correct type-hinting is still a good reason to use but as you said, there are other options there as well.

**Comment by shaffe-fr** [comment]
Hi, I had no out of the box issues on my project with about 20 factories.

**Comment by christophrumpel** [comment]
I just released v2 which **ONLY** works with Laravel 8. Everyone who wants to, can already upgrade.

I still need to rewrite the docs to give a better understanding of where the differences are to Laravel 8 factories.
